### ISSUE TITLE
question about dimp-nce+

### ISSUE THREAD

**Issue by lerohiso** [fregu856/ebms_regression#2]
I applied your NCE+ method to super_dimp.
As written in the paper, only the loss function was modified.
And both training and inference used super_dimp's method as it is.
However, it is performing worse than super_dimp.
(GOT-10K AUC - 0.801 vs 0.759)
Are there any other modifications not mentioned in the paper?
(Are the same values ​​for the loss weight and learning rate used as the super_dimp method?)
